fetch.bs

```diff
@@ -184,12 +184,74 @@ lt="authentication entry">authentication entries</a> (for HTTP authentication).
  <dt><dfn for="fetch params">process request end-of-body</dfn> (default null)
  <dt><dfn for="fetch params">process response</dfn> (default null)
  <dt><dfn for="fetch params">process response end-of-body</dfn> (default null)
+ <dt><dfn for="fetch params">process response done</dfn> (default null)
  <dd>Null or an algorithm.
 
  <dt><dfn for="fetch params">task destination</dfn> (default null)
  <dd>Null, a <a for=/>global object</a>, or a <a for=/>parallel queue</a>.
+ <dt><dfn for="fetch params">timing info</dfn>
+ <dd>A <a for=/>fetch timing info</a>.
+ <dt><dfn for="fetch params">redirect timing info list</dfn> (default « »)
+ <dd>A <a for=/>list</a> of <a for=/>fetch timing info</a>.
 </dl>
 
+<p><dfn>Unsafe shared current time</dfn> is defined in [[HR-TIME]].
+<p class=XXX>Remove this once HR-TIME-3 refs are available.
```

Comment:
You can remove that bit now

Reply:
I can't until Bikeshed sees HR-TIME-3

Reply:
I tried it locally, and it properly linked...